src/Symfony/Bundle/TwigBundle/DependencyInjection/Configuration.php

```diff
@@ -203,4 +204,16 @@ private function addTwigFormatOptions(ArrayNodeDefinition $rootNode)
             ->end()
         ;
     }
+
+    private function addEventDispatcher(ArrayNodeDefinition $rootNode)
+    {
+        $rootNode
+            ->children()
+                ->variableNode('event_dispatcher')
+                    ->info('The event dispatcher')
```

Comment:
not sure this is useful - can we add more info why this is worth configuring? if not, remove it?

Reply:
Also changed the message, is this what you have in mind?